### DISCUSSION TITLE
Problems with codedocs.xyz (DOxygen publication) integration

### DISCUSSION THREAD

**Discussion by DRNadler** [community/community#137874] [Programming Help]
### Select Topic Area

Question

### Body

About a zillion github repositories document code using https://www.doxygen.nl/
All of us want to publish the Doxygen HTML for other users, and don't want to put the generated HTML in our repos.
Eons ago, someone created the great [codedocs.xyz ](https://codedocs.xyz) tool to generate Doxygen HTML from a github repository and publish it, for example: https://codedocs.xyz/DRNadler/wxMathPlot/class_math_plot_1_1mp_layer.html

Unfortunately, this tool has a couple of problems:

1. It doesn't have the **dot** utility (part of the [Graphviz](https://graphviz.org/) package) installed, so the graphics normally generated by DOxygen are not generated, as in the missing inheritance graph in the example I linked above, and
2. It is using a truly ancient version of DOxygen

Does anyone know who is maintaining this tool and how we can get it updated?
This repository is empty and no one has replied to open issues: https://github.com/CodeDocs/CodeDocs

Thanks!
Best Regards, Dave

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @DRNadler ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
